scripts/recan_bundle_tdr.py

```diff
@@ -329,18 +353,33 @@ def main(argv):
     Load a canned bundle from DCP/1 and write *.manifest.tdr and *.metadata.tdr
     files showing the desired output for DCP/2.
     """
+    default_version = datetime(year=2021,
```

Comment:
This looks like it fits one line. If it does, it would look better. That's why I requested you extract the variable.